### ISSUE TITLE
Enforce username matching?

### ISSUE THREAD

**Issue by coredump** [keybase/keybase-issues#1781]
I know this may sound a little weird, but would not it be possible to enforce matching usernames to things maybe like twitter or github account?

Example: I am "coredump" in GH, twitter, facebook, but in keybase "coredump" is someone that goes by something totally different on all the services.

**Comment by MuhammedZakir** [comment]
Some people use different usernames for different sites. What about them? For example: Username in Github, Username1 in Twitter, Username in Facebook etc... Which name should we enforce? Honestly, this is not a good idea!

**Comment by koluke** [comment]
I agree with @MuhammedZakir.  Enforcing usernames becomes a tricky business especially because not everyone uses the same username across every platform.  I could see a feature being implemented possibly to warn users that there is a Keybase user with the same name as a Twitter, GitHub, or similar user, but that feature would be rendered unnecessary by the lack of a post or Gist verifying ownership of both the social media account and the PGP key in Keybase.
